src/plotter_disk.hpp

```diff
@@ -16,8 +16,11 @@
 #define SRC_CPP_PLOTTER_DISK_HPP_
 
 #ifndef _WIN32
+#include <pthread.h>
```

Comment:
thread support is standarized. you can include `<thread>` and have a portable `std::thread` class